source/manual/govuk-in-aws.html.md

```diff
@@ -1,7 +1,7 @@
 ---
 owner_slack: "#govuk-2ndline"
 title: GOV.UK in AWS
-section: AWS
+section: AWS architecture
```

Comment:
Given that there's an Infrastructure section, would it make sense to call this "AWS Infrastructure"?

Reply:
🤔 maybe it should move to a "Architecture" section?

Reply:
I've done another pass. I've created a new "Architecture" section with these pages in it, and I've moved some stuff from Basics in there too. To keep it focused I've renamed "Infrastructure" to "Infrastructure tasks".